github-actions/pulumi-dotnet.yml

```diff
@@ -27,17 +27,10 @@ env:
 jobs:
   pulumi:
     name: Pulumi
+    concurrency: [[ .PulumiStackName ]]
```

Comment:
Need to make this a conditional so only updates are throttled (previews shouldn't be).